dashboard/config/linux/bits/android-5.10.yml

```diff
@@ -10,7 +10,9 @@ kernel:
  tag: 8d21bcc704ea1
 
 shell:
- - make gki_defconfig
+ - make gki_defconfig: [-virtual_modules]
```

Comment:
Together with the `modules` introduced by this PR, the `virtual_modules` name looks like some fundamental architectural piece of the kernel. Maybe a more specific name would be better in this case?

Reply:
Makes sense! I changed it to "cuttlefish"